### DISCUSSION TITLE
Unable to write plex password

### DISCUSSION THREAD

**Discussion by Fryser1850** [Taxel/PlexTraktSync#1201] [Support] [answered]
Hello, I have a problem with the set up of my plex account. I can set my username/email but I can't type in the password. Can anyone help me?
Thanks in advance

**Comment by Fryser1850** [comment, marked as answer]
ok my bad... im so dumb... problem solved

**Reply by Aejor** [reply]
How did you solve this? I am apparently also dumb

**Reply by Fryser1850** [reply]
When you type your password it is written in real but you just don't see it.
I guess it's for security

**Reply by Aejor** [reply]
Thank you so much!

**Reply by Fryser1850** [reply]
Np have fun
